app/src/androidTest/java/tech/relaycorp/courier/domain/PrivateSyncTest.kt

```diff
@@ -30,6 +32,11 @@ class PrivateSyncTest {
 
     @Test
     fun privateSync() {
+        assumeTrue(
+            "Test is currently failing on API 23 and lower due to a grpc internal issue",
+            Build.VERSION.SDK_INT >= 24
+        )
```

Comment:
Can you please add a comment with a link to #584? I won't remember what this relates to in the next 5 ~days~ seconds 😬

Reply:
Done

Reply:
Thanks!